react-native-gutenberg-bridge/ios/RNReactNativeGutenbergBridge.swift

```diff
@@ -38,6 +38,19 @@ public class RNReactNativeGutenbergBridge: RCTEventEmitter {
         }
     }
 
+    @objc
+    func onCapturePhotoPressed(_ callback: @escaping RCTResponseSenderBlock) {
+        DispatchQueue.main.async {
+            self.delegate?.gutenbergDidRequestMediaFromCameraPicker(with: { (mediaID, url) in
+                guard let url = url, let mediaID = mediaID else {
+                    callback(nil)
+                    return
+                }
+                callback([mediaID, url])
```

Comment:
Just an idea:
All the media events in the bridge look super similar. (`onMediaLibraryPressed`. `onUploadMediaPressed`, `onCapturePhotoPressed`)
Maybe we could merge them and send a flag to identify them?

Reply:
Just proposed that on the Android PR that has the bridge method for JS. I can also change the delegate here to make it a single method.

Reply:
Here: https://github.com/WordPress/gutenberg/pull/13450#pullrequestreview-196011991

Reply:
Heh nice! 👍